package.json

```diff
@@ -61,7 +61,7 @@
     "yaml": "1.10.0"
   },
   "devDependencies": {
-    "@types/electron-devtools-installer": "2.2.0",
+    "@types/electron-devtools-installer": "^2.2.0",
```

Comment:
This and electron-devotols-installer have ^

Reply:
This is an intentional change. Devtools installer shouldn't affect the releases, but when it's outdated, some of its dependencies can change to make it fail to load.

This is more of a symptom of us not commiting package-lock.json than anything else though.

Reply:
electron-devtools-installer was getting broken by dependencies updating under it.

I'll revert it, but we should probably start commiting package-lock.json to prevent these things in the future. Probably best to do it during the next release branch.